client/src/components/config-card/config-card.tsx

```diff
@@ -175,8 +175,10 @@ export const ConfigCard: React.FC<IConfigCardProps> = ({ className }) => {
       downstreamUrl: config.downstreamUrl,
       id: config.id,
     };
-
-    await createSniffer(userId, saveConfig)
+    console.log(userId, saveConfig);
+    saveConfig["id"] = saveConfig["name"];
+    saveConfig["userId"] = userId;
```

Comment:
Don't need to add userId manually. As it is added automatically from the cookie.

Reply:
ok I did it since i don't success to create sniffer

Reply:
Ok you can remove it. I Fixed it in my branch :)